orbit/pkg/packaging/windows.go

```diff
@@ -222,7 +222,7 @@ func checkWine(wineChecked bool) error {
 		cmd := exec.Command(wix.WineCmd, "--version")
 		if err := cmd.Run(); err != nil {
 			return fmt.Errorf(
-				"%s failed. Is Wine installed? Creating a fleetd agent for Windows (.msi) requires Wine. To install Wine see the script here: https://github.com/fleetdm/fleet/blob/fleet-v4.44.0/scripts/macos-install-wine.sh %w",
```

Comment:
ideally we should have an url somewhere that redirects to the right place, eg: fleetdm.com/install-wine.

optionally, should we just do `main`?

Reply:
I can remove this. the change isn't required. Just my sed being a little ambitious.